tests/test_views.py

```diff
@@ -55,10 +61,11 @@ def test_format_error_with_two_locations():
     ]
 
 
-def test_graphqlresult_to_response():
+def test_graphqlresult_to_response(app): # noqa
```

Comment:
@jimrollenhagen here is where app gets injected ... it's not 'used' but just injecting it will force the fixture to yield inside of a request context ... which is the bare minimum of what we need here.